main.yaml

```diff
@@ -249,6 +253,7 @@ Metadata:
       - Label:
           default: "Chef Server Settings"
         Parameters:
+          - ChefServerCustomConfig
```

Comment:
Same request as above, let's stick this at the bottom of this section

Reply:
Moved!